ksql-parser/src/main/java/io/confluent/ksql/parser/tree/DefaultTraversalVisitor.java

```diff
@@ -362,4 +372,12 @@ protected R visitInsertInto(InsertInto node, C context) {
     process(node.getQuery(), context);
     return null;
   }
+
+  protected R visitCreateStreamAsSelect(CreateStreamAsSelect node, C context) {
```

Comment:
just curious - is this used anywhere? the analyzers seem to just operate on Query nodes.